test/files/rules/indentwith_tabs.test.ts

```diff
@@ -1,5 +1,8 @@
 // valid code
 module TestModule {
+	/**
+	 * Some documentation
+	 */
```

Comment:
Not with the current implementation, this just disables all indentation checking between `/*` and `*/`. This will allow #485 I believe, the test case rendered with `·` for spaces and `⇥` for tab is:

```
⇥/**
⇥·*·Some·documentation
⇥·*/
```

I would prefer to allow this via [smart-tabs](http://www.emacswiki.org/emacs/SmartTabs), but that would be a lot more work to implement.